app/src/util/constants.ts

```diff
@@ -1,31 +1,103 @@
-import { MdOutlineHomeWork } from "react-icons/md";
-import { FiTrash2, FiTruck } from "react-icons/fi";
+import { BsTruck } from "react-icons/bs";
+import { PiTrashLight } from "react-icons/pi";
+import { TbBuildingCommunity } from "react-icons/tb";
+import { PiPlant } from "react-icons/pi";
+import { IconBaseProps } from "react-icons";
+import { LiaIndustrySolid } from "react-icons/lia";
 
 export const maxPopulationYearDifference = 5;
 
-export const SECTORS = [
+export type InventoryType = "gpc_basic" | "gpc_basic_plus";
+export enum InventoryTypeEnum {
+  GPC_BASIC = "gpc_basic",
+  GPC_BASIC_PLUS = "gpc_basic_plus",
+}
+
+export interface ISector {
+  number: number;
+  referenceNumber: string;
+  icon: React.ElementType<IconBaseProps>;
+  description: string;
+  name: string;
+  testId: string;
+  inventoryTypes: {
+    [InventoryTypeEnum.GPC_BASIC]: {
+      scopes: number[];
+    };
+    [InventoryTypeEnum.GPC_BASIC_PLUS]: {
+      scopes: number[];
+    };
+  };
+}
+
+export const getSectorsForInventory = (inventoryType?: InventoryType) => {
+  if (!inventoryType) return [];
+  return SECTORS.filter((sector) => {
+    const scopesForInventoryType =
+      sector.inventoryTypes[inventoryType as InventoryType];
+    return scopesForInventoryType?.scopes.length > 0;
+  });
+};
+
+export const SECTORS: ISector[] = [
   {
-    sectorName: "stationary-energy",
+    referenceNumber: "I",
+    number: 1,
+    name: "stationary-energy",
+    description: "stationary-energy-description",
+    icon: TbBuildingCommunity,
+    inventoryTypes: {
+      [InventoryTypeEnum.GPC_BASIC]: { scopes: [1, 2] },
```

Comment:
Shouldn't this be 1 and 3 for most sectors in basic?

Reply:
that's for waste